### DISCUSSION TITLE
8.2.0.24 chrome downloads a couple/few videos and then just doesn't download any more until I exit and reopen browser

### DISCUSSION THREAD

**Discussion by dhcernese** [aclap-dev/video-downloadhelper#160] [Help and Q&A]
### Your question

It's been a few months since I used the app.  I have a subscription to paid training videos (legit), I download them so I can watch at my own schedule.. ..so even if I shared the link you can't login to test them.  Basically download _usually_ works, but stops working.

Using licensed chrome version 8.2.0.24
I use the pop-up and choose the resolution to download.  Usually this works, I have just today seen 2 or even 3 parallel downloads working.  After waiting for those to finish and go to the next video, the pop-up is there with the list of resolutions but nothing downloads.  Sometimes in task manager I can see a powershell pop-in and disappear.  The helper app is there.

Sometimes it even says there is no media to process. Refreshing does nothing; but if I close the tab and open it again, then it offers the resolutions to download, and usually starts.  Why do I have to keep existing my browser?  Is there any place I can look for logs or error messages?

**Comment by dhcernese** [comment]
Sometimes the pop-up is showing the information from the previous video and not the one on the screen.  or worse, it is not showing the actual resolutions and i can end up downloading a lower resolution than i wanted.

**Comment by dhcernese** [comment]
...and now, even after closing browser window and opening again.. ..it shows the proper video and resolution in the pop-up but the download does not start.  I've tried a few times; waiting between each (because it takes anywhere from a few seconds to 30 seconds for the file to start).  I can see the companion app time out and go away; I attempt a download and it comes back.. ..but no file shows up.

**Comment by dhcernese** [comment]
I found more reliable behavior if I used the settings => more => reload extension.  then reload the page.  it seems to be needed after a few downloads

**Comment by paulrouget** [comment]
>  but nothing downloads

what do you mean? It stays stuck at 0%? It fails?

> Addon & CoApp version and details
> No response

Please add details: https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F

**Reply by dhcernese** [reply]
What do you mean "stuck at 0%", there is no progress status anywhere.  The file just never starts being downloaded.

![image](https://github.com/aclap-dev/video-downloadhelper/assets/1350162/8c9d77df-33f4-4e0c-9ae1-65f29903f80a)

**Comment by paulrouget** [comment]
@dhcernese this worries me a little, as I don't see how this could keep failing.

So you click on the video size you want, then the popup it doesn't do anything at all? Like, it's like you didn't click? Or does the download start, but then ends right away? Do it say the download succeeded? It doesn't say `downloading - 1%` at any point?

**Reply by paulrouget** [reply]
Or does it go in "queue" mode?

**Reply by dhcernese** [reply]
I click the resolution and the popup goes away and nothing happens as if I didn't click it.  I even tried at different locations just in case there was a bug where I clicked it.  Something happens in the background because I saw a powershell script flash by in the task manager.

When you say "downloading - %' ...where would that appear?  I've downloaded 100 videos, I have no idea where I would see progress.  Normally I just watch the windows filesystem.. ..when the video is finished, windows adds the time length and I know it's done.

**Reply by dhcernese** [reply]
as I mentioned in another comment; if nothing happens in 30 seconds (that's usually how long it takes to start writing the file) -or- I notice the resolution or title is not the current video, I reload the extension and all starts good again.

**Reply by paulrouget** [reply]
You don't see a blue bubble in the popup after you start the download?
If so, click on the bubble, and tell me what you see.
If you don't, do you see a red bubble?

**Reply by dhcernese** [reply]
Where does this imaginary bubble show up?  
![image](https://github.com/aclap-dev/video-downloadhelper/assets/1350162/0a3014c4-ef54-4f0d-917b-9c77a4cca249)

**Reply by dhcernese** [reply]
btw, that's _after_ it was selected and in the process of downloading already.  I just did 3 videos. ..and now it does nothing again.

**Reply by GrampaWildWilly** [reply]
When I went to YouTube just now, this link was near the top.  Since it was short, I clicked it:

https://www.youtube.com/watch?v=_JDWVlMtthk

 
![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/c986c596-5b11-4282-8b32-209d487d2f7c)
![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/b98a10ca-221e-492a-becd-db9f299e9bf2)
![#03](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/e1e845b9-730d-4631-86c0-890eff86da79)

**Reply by dhcernese** [reply]
Ah! I never used that before.  So this time, I used that widget and same problem but with a clue.  After a few successes, I get this:
![image](https://github.com/aclap-dev/video-downloadhelper/assets/1350162/bb4dd083-01d2-4585-8db6-56f4fe01c661)

...and now the next video won't download.  I click, nothing happens.

![image](https://github.com/aclap-dev/video-downloadhelper/assets/1350162/c049c356-187c-43d2-9772-8c79c99c2b98)

**Reply by GrampaWildWilly** [reply]
No, that's not anything.  When you have multiple pages on which VDH detects something, and you close one of those pages, you'll see this orphan indication on one of the pages that is still open.  This is just an indication of how many items VDH had previously detected on the page you just closed.  This orphan indication goes away after a certain amount of time.  That time is controlled by a VDH setting.  Look on the Behavior tab of VDH settings.  You will find a setting named "Orphan expiration timeout."  I'm sorry I can't help you with your problem.  I don't use Chrome.  I will say that you are not on the latest release of everything.  I am running Windows 11 Pro 64-bit version 23H2 build 22631.3155, Firefox 123.0 64-bit, licensed VDH 8.2.1.3a1 beta, CoApp 2.0.19.  The significant tidbit in that is the release of the CoApp.  Also, you should know that I don't work for VDH.  I'm just another user.

**Comment by paulrouget** [comment]
Hooo! You use the context menu!

There was a problem indeed, which has been fixed in 8.2.1.

So… I think this is solved now?
